.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,4 +40,7 @@ or AWS CLI commands to trigger something.
 - [ ] Fix linting errors
```

Comment:
This will also cause Travis to fail so there might be no need to check it explicitly.